src/platform/packages/shared/kbn-coloring/src/color_manipulation/index.ts

```diff
@@ -29,3 +29,26 @@ export const getColorAlpha = (color?: string | null) =>
 
 export const makeColorWithAlpha = (color: string, newAlpha: number) =>
   chroma(color).alpha(newAlpha);
+
+export function getValidColor(color?: string | null): chroma.Color | undefined;
+export function getValidColor(
+  color?: string | null,
+  options?: { shouldBeCompatibleWithColorJs: true }
+): chroma.Color;
+export function getValidColor(
+  color?: string | null,
+  { shouldBeCompatibleWithColorJs }: { shouldBeCompatibleWithColorJs?: boolean } = {
+    shouldBeCompatibleWithColorJs: false,
```

Comment:
Instead of calling out the compatibility with a dependency that is not there anymore, isn't better to specify what actually this code is doing? defaulting invalid colors to a specific one?

domething like `defaultColor` and passing the actual default to it?

Reply:
I would see this as a 2 phases step: the current mantainers can quickly grasp here we're adding a compatibility layer.
We can do a follow up next where things get normalized.